amen/audio.py

```diff
@@ -18,4 +20,26 @@ def __init__(self, file_path, convert_to_mono=False, sample_rate=22050):
         self.raw_samples = y
         self.num_channels = y.ndim
         self.duration = librosa.get_duration(y=y, sr=sr)
+        self.features = self.create_features()
+
+    def create_features(self):
+        features = {}
+        features['centroid'] = self.get_centroid()
+        return features
+
+    def get_centroid(self):
+        """
+        Gets spectral centroid data from librosa and loads it into a feature
+        """
+        hop_length = 512
+        mono_samples = librosa.to_mono(self.raw_samples)
+        centroid = librosa.feature.spectral_centroid(mono_samples, hop_length=hop_length)
+        centroid = centroid[0]
+
+        indexes = range(len(centroid))
+        indexes = [((hop_length * i) / self.sample_rate) for i in indexes]
```

Comment:
[librosa.frames_to_time](http://bmcfee.github.io/librosa/generated/librosa.core.frames_to_time.html?highlight=frames_to_time#librosa.core.frames_to_time) does this directly.  Probably better for readability to use that instead.